llvm/include/llvm/Support/Allocator.h

```diff
@@ -22,6 +22,7 @@
 #include "llvm/Support/AllocatorBase.h"
 #include "llvm/Support/Compiler.h"
 #include "llvm/Support/MathExtras.h"
+#include "llvm/Support/raw_ostream.h"
```

Comment:
is this needed or was it added earlier for debugging?

Reply:
This was added for debugging. I removed it.